### PR TITLE
Adds a version of the CO armor without pauldrons

### PR DESCRIPTION
**Pull request by TheGamerdk**
# About the pull request

This is based on the old sprite, and is a stat-identical version of the normal CO armor, minus the pauldrons.

Nessie is behind this, please refer criticism that way. (Unless it's the code, that's on me)

# Explain why it's good for the game

More choices for drip baby!

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

image

Can only pick 1:
image

Worn:
image

</details>

# Changelog
:cl:
add: COs have a new variant of their armor to choose from
/:cl:

### CODE REVIEW

File: code/game/machinery/vending/vendor_types/crew/commanding_officer.dm [hunk 1]
```diff
@@ -82,8 +82,11 @@ GLOBAL_LIST_INIT(cm_vending_clothing_commanding_officer, list(
 		list("Commanding Officer Backpack", 0, /obj/item/storage/backpack/mcommander, MARINE_CAN_BUY_BACKPACK, VENDOR_ITEM_MANDATORY),
 		list("Secure Satchel", 0, /obj/item/storage/backpack/satchel/lockable, MARINE_CAN_BUY_BACKPACK, VENDOR_ITEM_MANDATORY),
 
-		list("COMBAT EQUIPMENT (TAKE ALL)", 0, null, null, null),
+		list("ARMOR (CHOOSE 1)", 0, null, null, null),
 		list("Commanding Officer's M3 Armor", 0, /obj/item/clothing/suit/storage/marine/MP/CO, MARINE_CAN_BUY_ARMOR, VENDOR_ITEM_MANDATORY),
+		list("Commanding Officer's M3 Armor (No Pauldrons)", 0, /obj/item/clothing/suit/storage/marine/MP/CO, MARINE_CAN_BUY_ARMOR, VENDOR_ITEM_MANDATORY),
```
Comment (Contrabang) [inline code/game/machinery/vending/vendor_types/crew/commanding_officer.dm:87]:
```suggestion
		list("Commanding Officer's M3 Armor (No Pauldrons)", 0, /obj/item/clothing/suit/storage/marine/MP/CO/no_pauldrons, MARINE_CAN_BUY_ARMOR, VENDOR_ITEM_MANDATORY),
```

Comment (TheGamerdk) [reply code/game/machinery/vending/vendor_types/crew/commanding_officer.dm]: i forgot to push my last commit, oops

**Review by Blundir — CHANGES_REQUESTED** [review]
I do not really support it for consistency reasons, if you want to make pauldron-less version, use the main CO armor for it, the previous officer armor was a placeholder, inform the spriter

**Comment by TheGamerdk** [comment]
> I do not really support it for consistency reasons, if you want to make pauldron-less version, use the main CO armor for it, the previous officer armor was a placeholder, inform the spriter

Updated the sprites. I can update the in-game example screenshots if you want, but seems unnecessary

**Comment by efzapa** [comment]
wouldn't it be better to just make it togglable like how you can roll down your uniform since there is no stat difference anyways

no need for item bloat here tbh

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by Blundir — APPROVED** [review]

**Pull request merged**